cmd/podman/main.go

```diff
@@ -119,6 +119,10 @@ func before(cmd *cobra.Command, args []string) error {
 	}
 	logrus.SetLevel(level)
 
+	if err := setSyslog(); err != nil {
```

Comment:
I think I beat you on this one in #3328

Reply:
I'll have to rework this.  Currently, I have an Info() message that will now be usually wrong. :)